lib/cog/request.rb

```diff
@@ -8,7 +7,7 @@ class Request
     def initialize
       @args = populate_args
       @options = populate_options
-      @input = JSON.parse(STDIN.read)
+      @input = (STDIN.tty?) ? {} : JSON.parse(STDIN.read)
```

Comment:
@chris if we were to want to pass something via STDIN during dev or testing would this change break that? And the bigger question, would we ever want to do that?